### ISSUE TITLE
Feature: Show NServiceBus version per endpoint

### ISSUE THREAD

**Issue by JSCProjects** [Particular/ServicePulse#869]
It would be really nice to have OOTB view about what version of NServiceBus you are running inside your organisation. Something like this

![image](https://user-images.githubusercontent.com/19533149/80206485-b0232e00-862c-11ea-86f5-6f6e0217d851.png)

**Comment by ramonsmits** [comment]
Nice suggestion @JSCProjects

**Comment by ramonsmits** [comment]
I took the liberty to enhance your proposal a bit to combine this with [support information](https://docs.particular.net/nservicebus/upgrades/supported-versions):

Don't verify the (sub)totals they probably don't add up correctly. The most important column I think is "Unsupported".

| Version | 5 | 6  | 7  | Total | Latest Minor | Latest | Expiration | Unsupported | Supported | Extended |
|---------|---|----|----|-------|--------------|--------|------------|-------------|-----------|----------|
| 5.2.21  | 1 |    |    | 1     | 5.2.26       | 7.2.4  | EOL        | 1           |           |          |
| 5.2.26  | 1 |    |    | 1     | 5.2.26       | 7.2.4  | 2020-10-11 |             |           | 1        |
| 6.2.1   |   | 9  |    | 9     | 6.2.8        | 7.2.4  | EOL        | 9           |           |          |
| 6.4.0   |   | 2  |    | 2     | 6.4.5        | 7.2.4  | EOL        | 2           |           |          |
| 6.4.2   |   | 1  |    | 1     | 6.4.5        | 7.2.4  | EOL        | 1           |           |          |
| 6.4.3   |   | 2  |    | 2     | 6.4.5        | 7.2.4  | EOL        | 2           |           |          |
| 6.4.5   |   | 1  |    | 1     | 6.4.5        | 7.2.4  | EOL        | 1           |           |          |
| 6.5.10  |   | 1  |    | 1     | 6.5.10       | 7.2.4  | 2020-05-29 |             | 1         |          |
| 7.0.1   |   |    | 6  | 6     | 7.0.3        | 7.2.4  | EOL        | 6           |           |          |
| 7.1.10  |   |    | 6  | 6     | 7.1.13       | 7.2.4  | EOL        | 6           |           |          |
| 7.1.13  |   |    | 6  | 6     | 7.1.13       | 7.2.4  | 2020-04-23 |             | 6         |          |
| 7.1.4   |   |    | 4  | 4     | 7.1.13       | 7.2.4  | EOL        | 4           |           |          |
| 7.2.0   |   |    | 3  | 3     | 7.2.4        | 7.2.4  | EOL        | 3           |           |          |
| 7.2.2   |   |    | 4  | 4     | 7.2.4        | 7.2.4  | EOL        | 4           |           |          |
| 7.2.3   |   |    | 22 | 22    | 7.2.4        | 7.2.4  | EOL        | 22          |           |          |
| 7.2.4   |   |    | 2  | 22    | 7.2.4        | 7.2.4  | Current    |             | 2         |          |
| Total   | 2 | 14 | 44 | 71    |              |        |            | 61          | 9         | 1        |

**Comment by TraGicCode** [comment]
I Agree, This would be very nice to have to have an organizational understanding of where your at in terms of versions.

**Comment by mikeminutillo** [comment]
Thanks all. I have added this to our feature request list and we'll update here when it gets prioritized.
